.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
   publish:
     docker:
-      - image: circleci/node:8.10.0
+      - image: circleci/node:11
```

Comment:
Might be an idea to bump this a bit higher while you're here.

Reply:
Hah maybe, I didn't want to change anything for you guys. Usually the lower the better in those tests, but we can setup a matrix as well, the way I did for laminar https://app.circleci.com/pipelines/github/ovotech/laminar?branch=0.11.0-release

But anyway this might be out of scope for this pr. Feel free to do those changes though.